index.html

```diff
@@ -47,79 +51,116 @@ <h2>Let's Connect</h2>
         </section>
         <section class="container">
             <div class="card">
-                <img class="first-image" src="/images/1.png" alt="">
+                <div id="first">
+                    <img src="images/61.png" alt="">
+                </div>
+                <div>
+            
                 <h2 class="title">Tonic</h2>
             
                     <ul class="field">
                         <li class="cano">CANOPY</li>
                         <li class="tag">&#8226; Back End Dev </li>
                         <li class="tag">&#8226; 2015</li>
                     </ul>
-                <p class="instruction"> A daily selection of privately personalized reads; 
+                    <p class="instruction">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                <p class="desk-praa"> A daily selection of privately personalized reads; 
                     no accounts or sign-ups required.</p>
+           
                 <ul class="trainning">
                     <li class="other-skills">html</li>
                     <li class="other-skills">css</li>
                     <li class="other-skills">javascript</li>
                 </ul>
                 <button class="card-btn">See Project</button>
             </div>
+        </div>
             <div class="card">
-                <img class="first-image" src="/images/2.png" alt="">
+            <div>
                 <h2 class="title">Multi-Post
                     Stories</h2>
             
                     <ul class="field">
-                        <li class="cano">CANOPY</li>
-                        <li class="tag">&#8226; Back End Dev </li>
+                        <li class="cano">FACEBOOK</li>
+                        <li class="tag">&#8226; Full Stack Dev </li>
                         <li class="tag">&#8226; 2015</li>
                     </ul>
```

Comment:
- [OPTIONAL] As I can't see this being pointed out by the previous reviewer, it seems like some CSS you introduced makes your mobile version deviate from the design, according to the Figma design, Your project image is meant to come before your project description on the mobile version of your portfolio, but that seems to be the opposite for the second and last project in your portfolio. 👍 Please you need to fix this as your final portfolio is meant to look great both in desktop and mobile. 😄
- - Your current design:
![Screenshot 2023-04-01 205842](https://user-images.githubusercontent.com/107632780/229312444-61d4f403-5960-4b49-afcb-608e1bc59cee.png)
- - Figma design:
![Screenshot 2023-04-01 205923](https://user-images.githubusercontent.com/107632780/229312455-0de55555-4209-4cef-9b75-668a026a33d7.png)